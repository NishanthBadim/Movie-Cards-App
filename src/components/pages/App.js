import React from 'react';
import Header from '../atoms/Header/Header';
import Movies from '../organisms/Movies';
import LoginPage from '../atoms/LoginPage'
import LogoutButton from './../atoms/LogoutButton'
import { useAuth0 } from '@auth0/auth0-react'
const App = () => {
    const { user, isAuthenticated } = useAuth0()
    //if (isAuthenticated)
        return (
            <div>
                <Header title="Movie Cards" />
                <div className="mt-4">
                    <Movies />
                </div>
            </div>
        );
    //else
      //  return <LoginPage />
}
export default App;