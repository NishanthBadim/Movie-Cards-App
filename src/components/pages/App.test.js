import React from 'react';
import {create} from 'react-test-renderer'
import "@testing-library/jest-dom/extend-expect";
import App from './App'
import { __esModule } from 'react-modal-video';
import { getByTitle, getByLabelText, render } from '@testing-library/react';

jest.mock('./../organisms/Movies', () => ({
    __esModule: true,
    default: () => { return <div></div>}
}))

jest.mock('./../atoms/Header/Header', () => ({
    __esModule: true,
    default: () => { return <div></div>}
}))


test("Match the snapshot - App",() => {
    const input=create(<App/>);
    expect(input.toJSON()).toMatchSnapshot();
})