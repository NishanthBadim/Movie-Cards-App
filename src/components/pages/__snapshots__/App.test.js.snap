// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Match the snapshot - App 1`] = `
<div>
  <div />
  <div
    className="mt-5"
  >
    <div />
  </div>
</div>
`;
